doc/manual/citing_aspect.bib

```diff
@@ -1450,3 +1451,88 @@ @article{LIU2022105509
+issn={2041-1723},
+doi={10.1038/s41467-022-33361-2},
+url={https://doi.org/10.1038/s41467-022-33361-2}
+}
+
+@article{https://doi.org/10.1029/2022GC010648,
+author = {Hollyday, Andrew and Austermann, Jacqueline and Lloyd, Andrew and Hoggard, Mark and Richards, Fred and Rovere, Alessio},
+title = {A revised estimate of early Pliocene global mean sea level using geodynamic models of the Patagonian slab window},
+journal = {Geochemistry, Geophysics, Geosystems},
+volume = {n/a},
+number = {n/a},
+pages = {e2022GC010648},
+keywords = {Geodynamics, Sea-level change, Pliocene, Glacial Isostatic Adjustment, Mantle Convection, Patagonia},
+doi = {10.1029/2022GC010648},
+url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2022GC010648},
+eprint = {https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2022GC010648},
+note = {e2022GC010648 2022GC010648},
+abstract = {Abstract Paleoshorelines serve as measures of ancient sea level and ice volume but are affected by solid Earth deformation including processes such as glacial isostatic adjustment (GIA) and mantle dynamic topography (DT). The early Pliocene Epoch is an important target for sea-level reconstructions as it contains information about the stability of ice sheets during a climate warmer than today. Along the southeastern passive margin of Argentina, three paleoshorelines date to early Pliocene times (4.8 to 5.5 Ma), and their variable present-day elevations (36 to 180 m) reflect a unique topographic deformation signature. We use a mantle convection model to back-advect present-day buoyancy variations, including those that correspond to the Patagonian slab window. Varying the viscosity and initial tomography-derived mantle buoyancy structures allows us to compute a suite of predictions of DT change that, when compared to GIA-corrected shoreline elevations, makes it possible to identify both the most likely convection parameters and the most likely DT change. Our simulations illuminate an interplay of upwelling asthenosphere through the Patagonian slab window and coincident downwelling of the subducted Nazca slab in the mantle transition zone. This flow leads to differential upwarping of the southern Patagonian foreland since early Pliocene times, in line with the observations. Using our most likely DT change leads to an estimate of global mean sea level of 17.5 ± 6.4 m (1σ) in the early Pliocene Epoch. This confirms that sea level was significantly higher than present and can be used to calibrate ice sheet models.}
+}
+ticle{https://doi.org/10.1029/2022GL099286,
```

Comment:
This seems to be broken!

Reply:
it does mystify me how these things happen.